cl/recap/mergers.py

```diff
@@ -1508,13 +1514,38 @@ def merge_attachment_page_data(
                 )
             except ValueError:
                 pass
-        rd.save()
+        await rd.asave()
 
         # Do *not* do this async — that can cause race conditions.
-        add_items_to_solr([rd.pk], "search.RECAPDocument")
+        await sync_to_async(add_items_to_solr)([rd.pk], "search.RECAPDocument")
+
+    de_query = RECAPDocument.objects.filter(
```

Comment:
Feels strange to call this the de_query. It's querying rds, right?

Reply:
changed to `rds`